map_topic: /map

robot_laser_topic: /robot0/laser_1

particles_number: 1000

enable_visualization: true

a1: 0.1

a2: 0.05

duration: 5
